generated/niscope/niscope.proto

```diff
@@ -473,8 +473,8 @@ enum ExportableSignals {
   EXPORTABLE_SIGNALS_NISCOPE_VAL_READY_FOR_ADVANCE_EVENT = 6;
   EXPORTABLE_SIGNALS_NISCOPE_VAL_READY_FOR_START_EVENT = 7;
   EXPORTABLE_SIGNALS_NISCOPE_VAL_READY_FOR_REF_EVENT = 10;
-  EXPORTABLE_SIGNALS_NISCOPE_VAL_5V_OUT = 13;
   EXPORTABLE_SIGNALS_NISCOPE_VAL_REF_CLOCK = 100;
+  EXPORTABLE_SIGNALS_NISCOPE_VAL_5V_OUT = 13;
```

Comment:
Note to self: Find whatever in hapigen is causing these to jump around. Guessing its some index equivalency which causes the ordering to fluctuate each build.